pkg/tfbridge/schema_test.go

```diff
@@ -1695,9 +1695,9 @@ func TestFailureReasonForMissingRequiredFields(t *testing.T) {
 				Schema: &ResourceInfo{
 					Tok: tokens.NewTypeToken("module", "importableResource"),
 					Fields: map[string]*SchemaInfo{
-						"inputY": {
```

Comment:
This is actually not right as map[string]*SchemaInfo is expected to refer to TF names such as input_y, so the test was incorrect and I believe the functionality also did not work as expected.